### DISCUSSION TITLE
How to encrypt specific notes?

### DISCUSSION THREAD

**Discussion by DeutscheGabanna** [tejado/obsidian-gpgCrypt#22] [Q&A]
There is no command in the command palette for anything starting with "encrypt" or "gpg". I do not want to encrypt all my files. New notes created after I set a key-pair for gpgCrypt are not encrypted either, nor do they have the .gpg extension I chose in the settings.

**Comment by DanMundy** [comment]
Adding my +1 (and thanks for your great work 🙂)

We can currently right-click on the tab and choose "Encrypt with key pair",
but I prefer to use keyboard for as many things as possible - so if we could have a command palette equivalent it would be really good. 
Thanks

**Comment by tejado** [comment]
Thats a very good point. Thanks for raising this - I will add it.

Feel free to provide an PR for this as well if you would like to implement this :)

**Comment by rubaboo** [comment]
I see this, did it not exist when you asked? If this is what you wanted, then the discussion can be marked Answered.

<img width="200" height="217" alt="image" src="https://github.com/user-attachments/assets/7e61cb9d-2494-4c5c-81ab-db43f56f21d6" />
